packages/lesmis-server/src/routes/ReportRoute.ts

```diff
@@ -25,10 +25,14 @@ export class ReportRoute extends TranslatableRoute {
         type: 'array',
         items: {
           type: 'object',
+          keys: 'all',
```

Comment:
I don't believe there's any reserved keys at this level, and since the response from a report varies wildly based on the report contents I thought a clean 'all' was better than combing through responses for all the possible key names that might exist at this point